lib/wrap-guide-element.coffee

```diff
@@ -89,6 +90,9 @@ class WrapGuideElement extends HTMLDivElement
     else
       @style.display = 'none'
 
+  setTooltip: ->
+    @title = "This line is the wrap-guide. You can quickly toggle it via the command palette."
```

Comment:
I wonder if people are going to know what to do with this. Something more explicit would be nice. Something short that tells them what the command palette is or how to invoke might be nice. 

> Toggle it by invoking the command palette (`cmd-shit-p`), and searching for wrap-guide.

They only need to know that stuff once though.